src/render/painter.js

```diff
@@ -251,6 +269,8 @@ class Painter {
             data: new Uint8Array([0, 0, 0, 0])
         }, context.gl.RGBA);
 
+        this.identityMat = mat4.identity(new Float32Array(16));
```

Comment:
I believe this is equivalent to gl-matrix [`mat4.create()`](https://github.com/toji/gl-matrix/blob/fccd5f707432ec5cc1774575be84c9f901a198d0/src/mat4.js#L13-L34)

```suggestion
        this.identityMat = mat4.create();
```

Reply:
Addressed in https://github.com/mapbox/mapbox-gl-js/pull/10564/commits/824fabd4f4544cef513ca5858d4692fd4abc830a